assets/models/system/Deci-DeciCoder-1b/spec.yaml

```diff
@@ -0,0 +1,31 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: Deci-DeciCoder-1b
+path: ./
+properties:
+  SHA: 59cead352e3b213c9a77097d3d08e89bf467ef7b
+  datasets: bigcode/starcoderdata
+  inference-recommended-sku: Standard_NC12s_v3, Standard_NC24s_v3, Standard_ND40rs_v2, Standard_ND96asr_v4, Standard_ND96amsr_A100_v4
+  evaluation-recommended-sku: Standard_NC12s_v3, Standard_NC24s_v3, Standard_ND40rs_v2, Standard_ND96asr_v4, Standard_ND96amsr_A100_v4
+  languages: EN
+  azureml.copyRegistryFilesToWorkspace: true
+tags:
+  Featured: ''
+  Preview: ''
+  inference_compute_allow_list:
+  - Standard_NC12s_v3
+  - Standard_NC24s_v3
+  - Standard_ND40rs_v2
+  - Standard_ND96asr_v4
+  - Standard_ND96amsr_A100_v4
+  evaluation_compute_allow_list:
+  - Standard_NC12s_v3
+  - Standard_NC24s_v3
+  - Standard_ND40rs_v2
+  - Standard_ND96asr_v4
+  - Standard_ND96amsr_A100_v4
+  SharedComputeCapacityEnabled: ''
+  task: text-generation
+  license: apache-2.0
+  author: deci-ai
```

Comment:
Are we going with deci-ai ? Can you confirm with Tina on this?

Reply:
Yes ayush ! Nancy and team has tested on this tag. Manoj was in loop always for this.